opm/material/fluidmatrixinteractions/EclHysteresisTwoPhaseLawParams.hpp

```diff
@@ -241,11 +339,20 @@ class EclHysteresisTwoPhaseLawParams : public EnsureFinalized
     void update(Scalar pcSw, Scalar /* krwSw */, Scalar krnSw)
     {
         bool updateParams = false;
-        if (pcSw < pcSwMdc_) {
+
+        if (config().pcHysteresisModel() == 0 && pcSw < pcSwMdc_) {
+            if (pcSwMdc_ == 2.0 && pcSw+1.0e-6 < Swcrd_ && oilWaterSystem_) {
+               initialImb_ = true;
```

Comment:
Are the results sensitiv to the 1.0e-6 here. I notice several hard-coded values like 1.0e-12 and 1.0e-8. It is a bit hard to evaluate the significance of these numbers.

Reply:
Sensitive: Yes, behavior will in general be different if we consider the process to be initial imbibition (following pure imb curve) compared to assuming reversal from drainage (following scanning curve). Whether the 1e-6 term is reasonable, or more general how to make the distinction between the two cases, can of course be debated.

Significance of 1e-N: Except from the case discussed above, the 1e-6 and 1e-8 are used to guard against division by zero, suggest to use 1e-8 everywhere. The 1e-12 is used once, and (as far as I remember) 1e-6**2 was chosen after some indications that 1e-6 was too large here.

Reply:
Thanks for the comment. I am ok with the current values.